AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
that file should be deleted from the depot

Reply:
deleted!